sencha-workspace/pages/src/page/StudentCompetencyDashboard.js

```diff
@@ -5,180 +5,31 @@ Ext.define('Site.page.StudentCompetencyDashboard', {
     requires: [
         'Site.Common',
         'Slate.cbl.model.Student',
-        'Slate.cbl.model.ContentArea',
-        'Slate.cbl.model.Competency',
+        'Slate.cbl.view.student.Dashboard',
+        'Ext.QuickTips',
         'Ext.XTemplate'
```

Comment:
@jmealo the XTemplate requirement can be removed now that this class isn't using it directly anymore. Looks like the student model isn't be used here anymore either